### PR TITLE
refactor: clean linter errors

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-753

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> One day they will all understand

**Review by RemiBonnet — APPROVED** [review]
LGTM expect for some duplicate imports, we can maybe add this rule on our linter https://eslint.org/docs/latest/rules/no-duplicate-imports

File: libs/pages/cluster/src/lib/feature/page-settings-advanced-feature/page-settings-advanced-feature.spec.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import { act, fireEvent } from '@testing-library/react'
+import { act, fireEvent } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/cluster/src/lib/feature/page-settings-advanced-feature/page-settings-advanced-feature.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

File: libs/pages/application/src/lib/ui/table-row-environment-variable/table-row-environment-variable.spec.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { getByText } from '@testing-library/react'
+import { getByText } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/ui/table-row-environment-variable/table-row-environment-variable.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

File: libs/pages/application/src/lib/ui/page-settings-storage/storage-modal/storage-modal.spec.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { findByLabelText, findByTestId, waitFor } from '@testing-library/react'
+import { findByLabelText, findByTestId, waitFor } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/ui/page-settings-storage/storage-modal/storage-modal.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

File: libs/pages/application/src/lib/feature/page-settings-storage-feature/page-settings-storage-feature.spec.tsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { findByTestId, waitFor } from '@testing-library/react'
+import { findByTestId, waitFor } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/feature/page-settings-storage-feature/page-settings-storage-feature.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

File: libs/pages/application/src/lib/ui/about/about-git/about-git.spec.tsx [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import { getByText } from '@testing-library/react'
+import { getByText } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/ui/about/about-git/about-git.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

File: libs/pages/application/src/lib/ui/import-environment-variable-modal/import-environment-variable-modal.spec.tsx [hunk 6]
```diff
@@ -1,4 +1,4 @@
-import { act, findAllByTestId, fireEvent, getByTestId, screen, waitFor } from '@testing-library/react'
+import { act, findAllByTestId, fireEvent, getByTestId, screen, waitFor } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/ui/import-environment-variable-modal/import-environment-variable-modal.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

File: libs/pages/application/src/lib/ui/job-overview/job-overview.spec.tsx [hunk 7]
```diff
@@ -1,4 +1,4 @@
-import { getAllByTestId, getByText } from '@testing-library/react'
+import { getAllByTestId, getByText } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/ui/job-overview/job-overview.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

File: libs/pages/application/src/lib/ui/page-settings-advanced/page-settings-advanced.spec.tsx [hunk 8]
```diff
@@ -1,4 +1,4 @@
-import { act, fireEvent, getByTestId } from '@testing-library/react'
+import { act, fireEvent, getByTestId } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/ui/page-settings-advanced/page-settings-advanced.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

File: libs/pages/application/src/lib/ui/page-settings-domains/page-settings-domains.spec.tsx [hunk 9]
```diff
@@ -1,4 +1,4 @@
-import { act, findByText, getByDisplayValue, getByRole, getByTestId, queryByTestId } from '@testing-library/react'
+import { act, findByText, getByDisplayValue, getByRole, getByTestId, queryByTestId } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/ui/page-settings-domains/page-settings-domains.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

File: libs/pages/application/src/lib/ui/page-settings-general/page-settings-general.spec.tsx [hunk 10]
```diff
@@ -1,4 +1,4 @@
-import { act, getByTestId, screen, waitFor } from '@testing-library/react'
+import { act, getByTestId, screen, waitFor } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/ui/page-settings-general/page-settings-general.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

File: libs/pages/application/src/lib/feature/import-environment-variable-modal-feature/import-environment-variable-modal-feature.spec.tsx [hunk 11]
```diff
@@ -1,4 +1,4 @@
-import { act, findAllByTestId, fireEvent, screen, waitFor } from '@testing-library/react'
+import { act, findAllByTestId, fireEvent, screen, waitFor } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/feature/import-environment-variable-modal-feature/import-environment-variable-modal-feature.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

File: libs/pages/application/src/lib/feature/page-settings-advanced-feature/page-settings-advanced-feature.spec.tsx [hunk 12]
```diff
@@ -1,6 +1,7 @@
-import { act, fireEvent } from '@testing-library/react'
+import { act, fireEvent } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/feature/page-settings-advanced-feature/page-settings-advanced-feature.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

File: libs/pages/application/src/lib/feature/page-settings-configure-job-feature/page-settings-configure-job-feature.spec.tsx [hunk 13]
```diff
@@ -1,4 +1,4 @@
-import { act, fireEvent, getAllByLabelText, getByLabelText, getByTestId, waitFor } from '@testing-library/react'
+import { act, fireEvent, getAllByLabelText, getByLabelText, getByTestId, waitFor } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/feature/page-settings-configure-job-feature/page-settings-configure-job-feature.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

File: libs/pages/application/src/lib/feature/page-settings-domains-feature/crud-modal-feature/crud-modal-feature.spec.tsx [hunk 14]
```diff
@@ -1,4 +1,4 @@
-import { act } from '@testing-library/react'
+import { act } from '__tests__/utils/setup-jest'
 import { render } from '__tests__/utils/setup-jest'
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/pages/application/src/lib/feature/page-settings-domains-feature/crud-modal-feature/crud-modal-feature.spec.tsx]: Fixed https://github.com/Qovery/console/pull/742/commits/82c1ad39d86568b9846b00aef57c94ea1e50ccc3

**Pull request merged**